d2bs/kolbot/libs/bots/BattleOrders.js

```diff
@@ -18,7 +55,11 @@ function BattleOrders() {
 					if (getTickCount() - tick >= failTimer * 1000) {
 						showConsole();
 						print("ÿc1BO timeout fail.");
-						quit();
+						if (Config.BattleOrders.QuitOnFailedGive) {
+							quit();
+						} else {
+							break;
```

Comment:
Why a break and not a return? (just wondering)

Reply:
I haven't written JS in a few years and ran into issues with the returns, where breaking out of the loop and letting the original code do the return worked. I'll revisit though and clean this up.